spec/controllers/api/v0/order_cycles_controller_spec.rb

```diff
@@ -80,6 +80,7 @@ module Api
             product2.update!(supplier: supplier)
             product3.update!(supplier: supplier)
             product4.update!(supplier: supplier, inherits_properties: false)
+            exchange.variants << product4.variants.first
```

Comment:
Why? Why don't you just ignore product4 and set `inherits_properties: false` on product3? Less code, less data, less to comprehend, less to compute.

Reply:
Yes, I thought about it yesterday, but I didn't have time to change it ; will do.